docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to Code Runner
+
+The following is a set of guidelines for contributing to Code Runner and its packages, which are a part of [Assignment-Exec Organization](https://github.com/assignment-exec) on GitHub.
+
+## Submit Changes or New Features
+
+### Clone the source code
+- To make changes to [code-runner](https://github.com/assignment-exec/code-runner) you need to first fork the repository and have the forked repository checked out locally.
+- After forking, copy the https url to clone the repository and use it in the command given below.
+```commandline
+$ git clone <https_url_of_repo>
+$ cd code-runner
+```
+
+### Make changes in a new branch
+Each change must be made in a separate branch, created from `develop` branch. Use git commands to create a branch and add changes.
+```commandline
+$ git checkout -b <branch_name>
+$ [edit files...]
+$ git add [files...]
+```
+
+### Commit changes
+- Write a clear log message for your commits. For small changes simple one-line messages are fine, but for big changes a detailed description should be provided.
+```commandline
+git commit
+```
+- First line in the commit messages should be a brief summary of the commit.
+- Follow the first line by a blank line and then the main description about the changes.
+- An ideal case would be that the commits are atomic i.e one feature or change is added per commit.
```

Comment:
It would be beneficial to have the commits atomic to ensure that the project has a cleaner history.

Reply:
Done